### PR TITLE
chore(deps): upgrade valibot

### PR DESCRIPTION
**Pull request by jamesgorrie**
Upgrades valibot. ~Hopefully~ A smaller filesize (-1.37 kB) and some new features.

### CODE REVIEW

File: support-frontend/assets/helpers/globalsAndSwitches/window.ts [hunk 1]
```diff
@@ -58,12 +58,15 @@ const PaymentConfigSchema = object({
 
 const ProductCatalogSchema = object({
 	productCatalog: record(
+		string(),
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/helpers/globalsAndSwitches/window.ts:61]: `productDescription`

File: support-frontend/assets/helpers/globalsAndSwitches/window.ts [hunk 2]
```diff
@@ -58,12 +58,15 @@ const PaymentConfigSchema = object({
...
 		object({
 			ratePlans: record(
+				string(),
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/helpers/globalsAndSwitches/window.ts:64]: `id`

File: support-frontend/assets/helpers/globalsAndSwitches/window.ts [hunk 3]
```diff
@@ -58,12 +58,15 @@ const PaymentConfigSchema = object({
...
 				object({
 					id: string(),
-					pricing: record(number()),
+					pricing: record(string(), number()),
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/helpers/globalsAndSwitches/window.ts:67]: `non-row region`

File: support-frontend/assets/helpers/globalsAndSwitches/window.ts [hunk 4]
```diff
@@ -58,12 +58,15 @@ const PaymentConfigSchema = object({
...
 					charges: record(
+						string(),
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/helpers/globalsAndSwitches/window.ts:69]: `Subscription/Contribution`

**Review by paul-daniel-dempsey — APPROVED** [review]
Looks good, just mapped up comments to properties just to aid my understanding

**Comment by jamesgorrie** [comment]
> to aid my understanding

If there's anything you'd like to run through - please let me know.

**Pull request merged**